specs/authorized_keys.table

```diff
@@ -0,0 +1,10 @@
+table_name("authorized_keys")
+description("A line-delimited authorized_keys table of per-user")
+schema([
```

Comment:
It would be nice to split each line on spaces and have columns for `n-1` = canonical/friendly name, `n-2` = public key data, `0 - n-3` = command/algorithm.